services/galley/src/Galley/API.hs

```diff
@@ -925,6 +926,17 @@ sitemap = do
         returns (ref Model.ssoTeamConfig)
         response 200 "SSO status" end
 
+    get "/custom-instance/by-domain/:domain" (continue CustomInstance.getCustomInstanceByDomainUntyped) $
+        capture "domain"
+        .&. accept "application" "json"
+
+    document "GET" "getCustomInstanceByDomain" $ do
+        summary "TODO"
+        parameter Path "domain" bytes' $
```

Comment:
```suggestion
        parameter Path "domain" text' $
```

Reply:
I actually reverted this suggestion as `text'` doesn't exist.

Reply:
sorry, i meant this: https://hackage.haskell.org/package/swagger-0.3.0/docs/Data-Swagger-Build-Api.html#v:string-39-